### PR TITLE
Issue 29 q and a

### LINKED ISSUE #29: block: FAQ
image

Content structure not definitively determined

### PR DESCRIPTION
**Pull request by fperez-leappoint**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #29 

Test URLs:
- Before: https://main--vonage--hlxsites.hlx.page/unified-communications/
- After: https://issue-29-q-and-a--vonage--hlxsites.hlx.page/drafts/qanda

### CODE REVIEW

File: blocks/qanda/qanda.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/* nothing yet */
```
Comment (synox) [inline blocks/qanda/qanda.css:1]:
```suggestion
```

File: blocks/qanda/qanda.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+.qanda-content > div::after {
+    background-color: #dcdde6;
+    border: 0;
+    height: .1rem;
+    margin-bottom: 3rem;
+    margin-top: 3rem
+}
+
+.qanda-content > div > label {
+    display: -webkit-box;
```
Comment (synox) [inline blocks/qanda/qanda.css:39]: the `-webkit-` prefixes are usually not needed. Please try without and test it.

**Review by synox — CHANGES_REQUESTED** [review]
looks great, just some minor comments

File: blocks/qanda/qanda.css (hunk 2 above)
Comment (fperez-leappoint) [reply blocks/qanda/qanda.css]: removed

File: blocks/qanda/qanda.css (hunk 1 above)
Comment (fperez-leappoint) [reply blocks/qanda/qanda.css]: forgot to take this one out, done

**Pull request merged**